apps/docs/content/components/CTAForm/index.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: CTA form
+description: Use CTA form to capture user input and call to action button.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image displaying a the CTA form anatomy, with a label on top of an input form, followed by a CTA button and optional checkbox underneath the input.](https://github.com/primer/brand/assets/131988618/2c86dd18-f3b0-41e3-a0d3-cca004ea37b1)
+
+- Label: Short text that describes the required input form. 
+- Text input: Single-line text field
+- Action: Call-to-action button that corresponds after completing the input form. 
+- Checkbox: An optional checkbox to request additional behavior.
+
+## Usage
+
+The CTA form is designed to capture user input and call to actions on a website. It serves as an efficient tool to prompt user engagement and action. While it can be placed throughout the page, it is encouraged to be positioned at the bottom to conclude the user journey, summarize key information, and encourage users to take action.
+
+The CTA form presents a concise label, prompting users to provide specific input in the form field. Once users enter the required information, they can trigger the call-to-action through the button. Additionally, the CTA form supports an optional checkbox to seek additional user behavior or acknowledge certain terms.
+
+By adopting the same guidelines as the respective [text input](/components/TextInput) component, [button](/components/Button) component, and [checkbox](/components/Checkbox) component, the CTA form ensures consistency in design and enhances user familiarity with the interface.
```

Comment:
non-blocker: I still think these two paragraph are a bit redundant and doesn't provide much usage guidelines to the consumer. The part of the checkbox is also repeated in the `Options` section.

I think we could re-phrase it a bit to remind that labels and buttons must be short and concise, and maybe linking to accessibility guidelines for buttons (https://primer.style/design/guides/accessibility/descriptive-buttons) or to our form patterns (https://primer.style/design/ui-patterns/forms/overview)